### DISCUSSION TITLE
Query all the Leetcode questions the users have done

### DISCUSSION THREAD

**Discussion by bytrangle** [alfaarghya/alfa-leetcode-api#11] [Q&A] [answered]
Hi. Thank you for making this useful API. I wonder if it is possible to query all the Leetcode problems a user has solved, not submissions. It seems like there is no direct way to do it. The closest thing seems to query `recentAcSubmissions` and filter the unique problem IDs. However, this query requires passing a `limit` variable 🤔. What do you think?

**Comment by alfaarghya** [comment, marked as answer]
Thank you 😊 @bytrangle .
- I am not sure if it's possible or not. However, I will try to find out. In the meantime, if you find a solution, please raise a PR, and I will be glad to merge it.

**Comment by Krishnateja001** [comment]
Workaround suggestion: We can use the last 20 submissions API to track activity for a specific user. However, since the API only returns recent submissions, we would need to run it as a scheduled cron job and continuously store the results in our database to maintain historical user data.
